src/index.ts

```diff
@@ -93,24 +101,16 @@ const makeSuffix = (noun: string, count = 1): suffixReturn => {
     schshxzNoun,
     standardNoun,
   ];
-  let result!: string;
 
-  if (typeof noun !== 'string' || noun === undefined) {
-    throw new TypeError('expected a string');
-  }
-
-  if (count === 1) return `${count} ${noun}`;
-  if (count < 0) return null;
-
-  for (let i = 0; i < nounFns.length; i += 1) {
-    const callFn = nounFns[i](noun, count);
+  for (const fn of nounFns) {
+    const callFn = fn(noun, count);
     if (callFn !== null) {
-      result = `${count} ${callFn}`;
-      break;
+      const result = `${count} ${callFn}`;
+      return result;
```

Comment:
Did you run the tests locally? Return will add the result and continue the loop where as break will end it as soon as it succeeds.